cluster/saltbase/salt/fluentd-gcp/init.sls

```diff
@@ -1,3 +1,4 @@
+{% if grains['roles'][0] == 'kubernetes-master' and grains.kubelet_api_servers is not defined -%}
```

Comment:
is `grains['roles'][0] == 'kubernetes-master'` redundant since this is only referenced in the kubernetes-master role in top.sls?